network/test/middleware_test.go

```diff
@@ -261,6 +261,35 @@ func (m *MiddlewareTestSuite) TestMaxMessageSize_SendDirect() {
 	require.Error(m.Suite.T(), err)
 }
 
+// TestLargeMessageSize_SendDirect asserts that a large message can be unicasted successfully
```

Comment:
I think the godoc is better to reflect more on the content of the test, we evaluate that a specific type of message that is our desire to be sent successfully although its size is beyond the limit.